data/mods/Magiclysm/field.json

```diff
@@ -11,5 +11,12 @@
     "half_life": "1 seconds",
     "phase": "solid",
     "display_field": false
+  },
+  {
+    "id": "fd_darkness",
+    "type": "field_type",
+    "intensity_levels": [ { "name": "shadow", "light_override": 3.7 } ],
```

Comment:
```suggestion
    "intensity_levels": [ { "name": "darkness", "light_override": 0.0 } ],
```

Reply:
Dunno how I missed that when I re-copied it.